bbb-common-web/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2'
+
+services:
+  postgres:
+    image: postgres:latest
+    container_name: postgres
+    environment:
+      - "TZ=America/Toronto Canada/Eastern"
```

Comment:
Should use the host machine `TZ`

Reply:
We should use `TZ=UTC` in the container, nothing else makes sense. Conversions from UTC to local time should happen in the application, not the database (if done at all).